### ISSUE TITLE
Parametrisize VESC can ID's

### ISSUE THREAD

**Issue by zerodamage** [SubC-Partner/uavcan_ros_bridge#1]
The VESC CAN'ids could be a ROS parameter here:

https://github.com/SubC-Partner/uavcan_ros_bridge/blob/16257c8cab1151528036a4c232c1e8363c7297c0/uavcan_ros_bridge/src/ros_to_uavcan_bridge.cpp#L54-L62

**Comment by jbrsc** [comment]
@zerodamage  Here exists a problem, each of ros_to_uav:: invokes a different RPM.CPP instance. These do not communicate, and after each successful run, they overwrite each others parameters. 
Meaning for id 106 an array of [1000,0,0,0,0,0,0,0,0,0] is returned while at 110 an array of [0,0,0,0,1000,0,0,0,0,0]
thereby affecting each other. (instead of running inside the same script)

Thereyby it might be necessary to only have 1 single ros_to_uav::conversionserver, which should include or pass an entire array of thruster inputs to the server?

**Comment by zerodamage** [comment]
Exactly, the change can be made in two ways, subscribe to all thrusters inside UAVCAN and collect the array (dirty) or change the thruster_manager so it also published the array directly (neat)

Best Regards/Venlig Hilsen
Christian Mai
________________________________
